config/initializers/simple_form.rb

```diff
@@ -6,7 +6,7 @@
   # stack. The options given below are used to wrap the
   # whole input.
   config.wrappers :default, class: :input,
-    hint_class: :field_with_hint, error_class: :field_with_errors do |b|
+                            hint_class: :field_with_hint, error_class: :field_with_errors do |b|
```

Comment:
Don't vertically align tokens on consecutive lines.
Reason: when you change one line, which requires you to re-align 5 other lines => it adds unnecessary noise to a PR.

Ref: https://github.com/thoughtbot/guides/blob/5df1e8fcc2166914b906f2284fb2719516679d3b/style/README.md#formatting

Reply:
Mmmm, but I would have done it to get under the 80 char minimum length. How do you choose which is more important? (And yes if I moved one I should have moved all)

Reply:
It's ok to break to multiple lines, but just indent normally (2 spaces)

Reply:
Got this fixed with:
Layout/AlignParameters:
  EnforcedStyle: with_fixed_indentation 

otherwise Rubocop wanted to violate the "don't vertically align tokens on consecutive lines"